src/pages/Dashboard/Dashboard.tsx

```diff
@@ -0,0 +1,18 @@
+import { useNavigate } from "react-router-dom";
+
+const Dashboard: React.FC = () => {
+  const navigate = useNavigate();
+  return (
+    <div>
+      <h1>Dashboard</h1>
+      <button
+        className="p-2 mt-2 text-white transition-colors rounded-md bg-slate-500 hover:bg-slate-400"
+        onClick={() => { navigate('/'); }}
```

Comment:
```suggestion
        onClick={() => navigate('/')}
```

Reply:
I'll commit this but the 'dashboard' component itself is mostly just a proof of concept for the auth